queries/omen/types.ts

```diff
@@ -2339,6 +2339,7 @@ export type FpmmTransaction = {
   __typename?: 'FpmmTransaction';
   additionalSharesCost?: Maybe<Scalars['BigInt']['output']>;
   collateralTokenAmount: Scalars['BigInt']['output'];
+  outcomes?: Maybe<Array<Scalars['String']['output']>>;
```

Comment:
Did you add this manually?

Reply:
Yes! I copied the type from the trades, but we can delete this as we're using the Trades only. Thanks for the heads up!